roles/installer/defaults/main.yml

```diff
@@ -164,7 +164,11 @@ ee_resource_requirements:
   requests:
     cpu: 500m
     memory: 1Gi
-
+    
```

Comment:
The yamllint is complaining about the trailing spaces on this line.  

```
$ yamllint .
./installer/defaults/main.yml
  167:1     error    trailing spaces  (trailing-spaces)
```

Other than that, this PR looks good to me.  Once we get CI passing, we should be able to merge this.  @Ghost0shell , do you mind pushing a fix for that?

Reply:
Hi, sorry for being late, i fixed that :)